internal/provisioner/objects/configmap/configmap.go

```diff
@@ -150,6 +143,8 @@ accesslog-format: envoy
 #   Configure the number of additional ingress proxy hops from the
 #   right side of the x-forwarded-for HTTP header to trust.
 #   num-trusted-hops: 0
+# Name of the envoy service to inspect for Ingress status details.
```

Comment:
ah, one thing i forgot to check on from the previous PR was if this was synced with what we have in the example configmap vs. still the string from the operator which might be out of date

hopefully this problem goes away when we use just the contourconfiguration CRD

Reply:
You mean all the commented out example config? Yeah, it is definitely missing some things, but yeah this will switch over to using ContourConfiguration instead of the ConfigMap pretty soon (I have some WIP code for it already just need the defaulting/validation revisions to land first) so I'm not worrying about it.

Reply:
> You mean all the commented out example config

yep

> this will switch over to using ContourConfiguration instead of the ConfigMap pretty soon

👍🏽 love it, sounds good